src/serve_data.js

```diff
@@ -245,6 +388,7 @@ export const serve_data = {
       const mbw = await openMbTilesWrapper(inputFile);
       const info = await mbw.getInfo();
       source = mbw.getMbTiles();
+      info['encoding'] = params['encoding'];
```

Comment:
Set this as tileJSON so it can be overwritten if supplied in mbtiles metadata.
```suggestion
      tileJSON['encoding'] = params['encoding'];
```

Reply:
Also, please add encoding to the pmtiles section above it.

ex.
```
      const metadata = await getPMtilesInfo(source);

      tileJSON['encoding'] = params['encoding'];
      tileJSON['name'] = id;
      tileJSON['format'] = 'pbf';
```